src/hooks/api/actions/usePlayground/playground.model.ts

```diff
@@ -1,91 +1,143 @@
-// TODO: replace with auto-generated openapi code
+import { VariantSchema } from 'openapi';
+import { Operator } from 'constants/operators';
 
-export enum PlaygroundFeatureSchemaVariantPayloadTypeEnum {
-    Json = 'json',
-    Csv = 'csv',
-    String = 'string',
-}
-
-export interface PlaygroundFeatureSchemaVariantPayload {
+export interface PlaygroundConstraintSchema {
```

Comment:
Could we use the regular OpenAPI code generation instead? 

If the playground schema uses features not supported by the generator, we could consider changing the schema instead. Either by using different/more fields, or relaxing the types that we can't express.

Would be great to support many generators, so that the spec is easy to use.

Reply:
I think it is a matter of the OpenApi generators (typescript-fetch specifically) catching up with the OAS 3.0.1 spec (anyOf, oneOf, allOf) are not generating nicely). Also 2 files seem to have missing functions (that should have been generated see DateSchema, PlaygroundFeatureProject is think) 'typescript-axios` generates fine btw.

I think we follow the specification here, not what the generator(s) can handle at this point in time. 

Looking at the openapitools-openapi-generator typescript-fetch docs : full support for anyOf, oneOf and allOf planned in short term roadmap

Reply:
Makes sense. Maybe we could do something like this:

1. Move this model file to `interfaces`, just to match other type files.
2. Add a comment to the top of the file about why it's here and what's been changed manually.
3. Link to the `typescript-fetch` issue for a later follow-up.